impeller/entity/contents/solid_color_contents.cc

```diff
@@ -66,15 +61,21 @@ bool SolidColorContents::Render(const ContentContext& renderer,
   }
 
   options.primitive_type = geometry_result.type;
-  cmd.pipeline = renderer.GetSolidFillPipeline(options);
-  cmd.BindVertices(std::move(geometry_result.vertex_buffer));
+
+#ifdef IMPELLER_DEBUG
```

Comment:
I think this can be automated away by performing this ifdef in `SetCommandLabel`?

Reply:
I think if we make this accept a string_view, then there won't be any copy if it is passed a const char*, and we can ifdef out before setting the label.